src/exprestive.coffee

```diff
@@ -89,7 +89,7 @@ class Exprestive
 
   # Save a function to @reversePaths to get a url back from a route name
   registerReverseRoute: ({routeName, url}) ->
-    @reversePaths[camelCase routeName] = (id) ->
+    @reversePaths[routeName] = (id) ->
```

Comment:
When I do a singular `as`, I think it would weird if you automatically made it available in the format I provided + camel case + snake case.

For the routes added by `resources`, you could add an overall configuration that lets the user choose between snake_case and camelCase with whichever default your prefer.

I don't think this really needs to be done as I don't know of any libraries that have this configuration but its certainly possible. Either way I think that can be a separate PR.